packages/blockchain/src/blockchain.ts

```diff
@@ -981,44 +981,48 @@ export class Blockchain implements BlockchainInterface {
       let blocksRanCounter = 0
       let lastBlock: Block | undefined
 
-      while (maxBlocks !== blocksRanCounter) {
-        try {
-          let nextBlock = await this.getBlock(nextBlockNumber)
-          const reorg = lastBlock
-            ? !equalsBytes(lastBlock.hash(), nextBlock.header.parentHash)
-            : false
-          if (reorg) {
-            // If reorg has happened, the _heads must have been updated so lets reload the counters
-            headHash = this._heads[name] ?? this.genesisBlock.hash()
-            headBlockNumber = await this.dbManager.hashToNumber(headHash)
-            nextBlockNumber = headBlockNumber + BigInt(1)
-            nextBlock = await this.getBlock(nextBlockNumber)
-          }
-          this._heads[name] = nextBlock.hash()
-          lastBlock = nextBlock
-          if (releaseLockOnCallback === true) {
-            this._lock.release()
-          }
+      try {
```

Comment:
lots of diff here is because of formatting change introduced because of this `try`